terraform/examples/ecs-complete/main.tf

```diff
@@ -0,0 +1,113 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "polytomic-ecs" {
+  source = "../../modules/ecs"
+
+  prefix = "polytomic"
+  tags = {
+    Owner       = "polytomic"
+    Environment = "staging"
+    Billing     = "R&D"
+  }
+
+  region = "us-east-1"
+
+  ####### Polytomic settings #######
+  polytomic_image = "568237466542.dkr.ecr.us-west-2.amazonaws.com/polytomic-onprem:rel2022.06.21.01"
+
+  polytomic_root_user      = "user@example.com"
+  polytomic_deployment     = "DEPLYOMENT"
+  polytomic_deployment_key = "DEPLYOMENT_KEY"
+
+  polytomic_google_client_id     = "GOOGLE_ID"
```

Comment:
Let's chat with James P this morning and figure out what the incantation looks like for SSO.